icssc/AntAlmanac
# Remove backend

## Summary
Now that the backend is available at `dev.api.antalmanac.com`, we can use that instead of running it locally! This drastically simplifies the onboarding experience for new developers. Devs no longer need to recursively clone the project or have the .env file for the backend.

In this PR, we do the following:
- remove the backend submodule
- move the contents of `/client` into the root directory
- update package.json in the root directory to be for the website
- update the readme accordingly

## Test Plan
- Delete package-lock.json, delete /node_modules, and re-run `npm i`
- Run `npm start` and verify that that site loads on `localhost:3000`. 
- Verify that I can load and save schedules to the `dev.api.antalmanac.com` backend.
- Run `npm run deploy` and verify that deployment succeeds

## Issues
#183  

## Future Followup
- Maybe make it easier to hook up the backend with the frontend for local development? Right now it requires a code change but it would be nice if it was just a flag or env variable.
